src/metabase/task/follow_up_emails.clj

```diff
@@ -66,7 +66,7 @@
 (def ^:private follow-up-emails-trigger-key "metabase.task.follow-up-emails.trigger")
 
 (defmethod task/init! ::SendFollowUpEmails [_]
-  (let [job     (jobs/build
+  (let [job    (jobs/build
```

Comment:
tiny nit: is there a reason for this whitespace change? it's no longer in line with the expression below

Reply:
I'll fix that. I probably did this to trigger the linting again :).

Reply:
Fixed.